cli/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 
 _Released 03/1/2023 (PENDING)_
 
+**Features:**
+
+- It is now possible to set `hostOnly` cookies with `cy.setCookie` for a given domain. Addresses [#16856](https://github.com/cypress-io/cypress/issues/16856).
```

Comment:
needs link to to `cy.setCookie()` command and also this addresses https://github.com/cypress-io/cypress/issues/17527
```suggestion
- It is now possible to set `hostOnly` cookies with `cy.setCookie()` for a given domain. Addresses [#16856](https://github.com/cypress-io/cypress/issues/16856).
```

Reply:
Addressed in a73e543359 and linked #17527 in the PR and in the issue